### PR TITLE
live777

### PR DESCRIPTION
**Pull request by Yin-ZP**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/126777)

### CODE REVIEW

File: manifests/y/yzp/live777/0.2.0/yzp.live777.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+PackageVersion: 0.2.0
+PackageLocale: en-US
+Publisher: yzp
+PublisherUrl: https://github.com/binbat/live777/releases/download/v0.2.0/live777-v0.2.0-i686-pc-windows-msvc.exe
+PublisherSupportUrl: https://github.com/binbat/live777/issues
+# PrivacyUrl:
+Author: yzp
+PackageName: live777
+PackageUrl: https://github.com/binbat/live777
+License: Apache-2.0, MIT
```
Comment (Trenly) [inline manifests/y/yzp/live777/0.2.0/yzp.live777.locale.en-US.yaml:14]:
```suggestion
License: MPL-2.0
```

File: manifests/y/yzp/live777/0.2.0/yzp.live777.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: yzp.live777
+PackageVersion: 0.2.0
+PackageLocale: en-US
+Publisher: yzp
+PublisherUrl: https://github.com/binbat/live777/releases/download/v0.2.0/live777-v0.2.0-i686-pc-windows-msvc.exe
+PublisherSupportUrl: https://github.com/binbat/live777/issues
+# PrivacyUrl:
+Author: yzp
```
Comment (Trenly) [inline manifests/y/yzp/live777/0.2.0/yzp.live777.locale.en-US.yaml:11]:
```suggestion
# Author:
```

File: manifests/y/yzp/live777/0.2.0/yzp.live777.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=NVS1.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: yzp.live777
+PackageVersion: 0.2.0
+PackageLocale: en-US
+Publisher: yzp
```
Comment (Trenly) [inline manifests/y/yzp/live777/0.2.0/yzp.live777.locale.en-US.yaml:7]:
```suggestion
Publisher: binbat
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/y/yzp/live777/0.2.0/yzp.live777.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=NVS1.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: yzp.live777
```
[the remark this answers is not in the document]
Comment (Yin-ZP) [reply manifests/y/yzp/live777/0.2.0/yzp.live777.installer.yaml]: can you help me with my latest version live777? It stop at stage "WinGetSvc-Validation (Installer Validation)".I just add a x64 version in yaml file.

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\y\yzp\live777\0.2.0', 
expected 'manifests\b\binbat\live777\0.2.0'

**Comment by stephengillie** [comment]
Hi @Yin-ZP,

The file names and path have to match the PackageIdentifier. Here they still reflect the previous PackageIdentifier. Please move these files out of `manifests\y\yzp\live777\0.2.0` and into `manifests\b\binbat\live777\0.2.0`.

Or if you give me your permission, I'd be happy to update this through GitHub's Editor.

**Comment by Yin-ZP** [comment]
Thank you for your help. I have sent you an invitation.
In my repository, it seems that the manifests\y\yzp\live777\0.2.0 have been deleted and put into the manifests\b\binbat\live777\0.2.0


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "microsoft/winget-pkgs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月23日(星期四) 凌晨2:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [microsoft/winget-pkgs] live777 (PR #126777)





 
Hi @Yin-ZP,
 
The file names and path have to match the PackageIdentifier. Here they still reflect the previous PackageIdentifier. Please move these files out of manifests\y\yzp\live777\0.2.0 and into manifests\b\binbat\live777\0.2.0.
 
Or if you give me your permission, I'd be happy to update this through GitHub's Editor.

**Comment by Yin-ZP** [comment]
I have sent an invitation to you. Have you received it? I see that the recent project has not been updated. Am I not successful in inviting?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "microsoft/winget-pkgs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月23日(星期四) 凌晨2:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [microsoft/winget-pkgs] live777 (PR #126777)





 
Hi @Yin-ZP,
 
The file names and path have to match the PackageIdentifier. Here they still reflect the previous PackageIdentifier. Please move these files out of manifests\y\yzp\live777\0.2.0 and into manifests\b\binbat\live777\0.2.0.
 
Or if you give me your permission, I'd be happy to update this through GitHub's Editor.

**Comment by stephengillie** [comment]
I was away for the American holiday. Sorry for the delay.

**Pull request closed without merging**